src/main/java/uk/gov/hmcts/reform/prl/services/managedocuments/ManageDocumentsService.java

```diff
@@ -277,10 +304,52 @@ public void moveDocumentsToRespectiveCategoriesNew(QuarantineLegalDoc quarantine
             );
             List<Element<QuarantineLegalDoc>> existingCaseDocuments = getQuarantineDocs(caseData, userRole, true);
             existingCaseDocuments.add(element(finalConfidentialDocument));
+            if (finalConfidentialDocument.getRespondentApplicationDocument() != null) {
+                Document document = finalConfidentialDocument.getRespondentApplicationDocument();
+                Element<PartyDetails> applicant = caseData.getApplicants().get(0);
+                if (!StringUtils.isEmpty(applicant.getValue().getEmail())) {
+                    emailAddress = applicant.getValue().getEmail();
+                    applicantName = String.format("%s %s",  applicant.getValue().getFirstName(),  applicant.getValue().getLastName());
+                }
+                Element<PartyDetails> respondent = caseData.getRespondents().get(0);
```

Comment:
line 314, will not work for FL401 case. so technically we need a check somewhere before.